src/client/components/NavigationalButtonComponent/NavigationalButtonComponent.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { withKnobs, text } from '@storybook/addon-knobs/react';
+import NavigationalButtonComponent from './NavigationalButtonComponent';
+
+export default {
+  title: 'NavigationalButton Component',
+  decorators: [withKnobs],
+};
```

Comment:
Hi Bengü! 
You seemed to have use knobs but when I run it in storybook I can't see them. Is this working?